e2e-polybft/framework/test-cluster.go

```diff
@@ -453,6 +486,12 @@ func (c *TestCluster) WaitUntil(dur time.Duration, handler func() bool) error {
 	}
 }
 
+func (c *TestCluster) WaitForReady(t *testing.T) {
```

Comment:
Do you need this function? Seems like you can just call `WaitForBlock` in test.

Reply:
So we always do an initial wait for block to make sure everything is ready, instead of writing `WaitForBlock(n)` this abstract that into its own concept.